<ui version="4.0" >
 <class>configInfo</class>
 <widget class="QDialog" name="configInfo" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Свойство конфигурации</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>20</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>131</y>
     <width>451</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string>Комментарии</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="commentEdit" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>151</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>124</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="text" >
     <string>Дата</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit" />
  </widget>
  <widget class="QSplitter" name="splitter_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>361</width>
     <height>97</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label" >
    <property name="font" >
     <font>
      <family>Arial KOI-8</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text" >
     <string>Наименование конфигурации:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editInfo" />
   <widget class="QLabel" name="label_2" >
    <property name="text" >
     <string>Автор</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="autorEdit" />
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>configInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>configInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
